ios/brave-ios/Sources/Brave/Frontend/Browser/UserScriptManager.swift

```diff
@@ -23,6 +23,7 @@ class UserScriptManager {
       .faviconFetcher,
       .rewardsReporting,
       .resourceDownloader,
+      .nightMode,
```

Comment:
Since we are injecting the script regardless of whether user enabled NightMode or not, did you do any performance tests? the script is 300kB

Reply:
The current night-mode script is always injected as well. It's web packed into the document start scripts.

I haven't noticed any performance issues at all. The script is also not enabled even though it's injected. It's only enabled via the toggle which calls the enable function, and only then it enables the style sheets.

I'll do some perf tests as well.

Reply:
>The current night-mode script is always injected as well. It's web packed into the document start scripts

the old script is 150 lines vs almost 8000 in the new one :)

>I'll do some perf tests as well.

Thanks!

Reply:
Haven't noticed any performance issues after a lot of testing. There doesn't seem to be any changes in loading speed of a page.

I think this is expected because the wallet scripts are around 85,000+ lines of JS and the pages still load fast (x2 because we load 2 files of that size), compared to this file.

The safari performance measurement in the debug tools don't seem to show any difference in load speeds :)